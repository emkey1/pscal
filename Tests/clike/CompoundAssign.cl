int main() {
    int x = 5;
    x += 3;  printf("%d\n", x);
    x *= 2;  printf("%d\n", x);
    x |= 1;  printf("%d\n", x);
    x -= 4;  printf("%d\n", x);
    x &= 15; printf("%d\n", x);
    return 0;
}
