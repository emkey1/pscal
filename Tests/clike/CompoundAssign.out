8
16
17
13
13
